en/docs/get-started/about-this-release.md

```diff
@@ -12,227 +12,58 @@ For more information on WSO2 API Manager, see the [overview]({{base_path}}/getti
-
-    You can now perform more advanced configurations for tenant configuration migration from the registry to the database.
-
-    **[Learn more]({{base_path}}/administer/advanced-configurations/)**
-
-??? note "File Inbound Endpoint subdirectory processing capability"
-
-    VFS Inbound Endpoints are capable of handling files inside subdirectories of the specified URL. This enables users to process files recursively inside a directory.
-
-    **[Learn more]({{base_path}}/reference/synapse-properties/inbound-endpoints/polling-inbound-endpoints/file-inbound-endpoint-properties/)**
-
-??? note "Improve Service Catalog to work with Hot Deployments"
-
-    You can now publish services to Service Catalog for hot deployments and include metadata files of the selected artifacts.
-
-    **[Learn more]({{base_path}}/tutorials/integration-tutorials/service-catalog-tutorial/)**
-
-??? note "Support to change lifecycle state of API Products using apictl"
-
-    You can now change the status of an API or API Product in an environment using apictl.
-
-    **[Learn more]({{base_path}}/install-and-setup/setup/api-controller/managing-apis-api-products/managing-apis-and-api-products/#change-status-of-an-api-or-api-product-in-an-environment)**
-
-??? note "Exposing proxy services via the Service Catalog"
-
-    This feature generates metadata artifacts for the APIs and Proxy services that have been created inside older Integration Studio workspaces.
-
-    **[Learn more]({{base_path}}/integrate/develop/generate-service-catalog-metadata/)**
-
-??? note "Support for SMB and WebDAV protocols"
-
-    This feature adds support to read from or write to files opened via SMB and WebDAV protocols.
-
-    **[Learn more](https://siddhi-io.github.io/siddhi-io-file/)**
-
-??? note "Support Protobuf map-type for Kafka"
-
-    This improvement adds support to decode Protobuf messages via a schema registry with Kafka.
-
-    **[Learn more](https://siddhi-io.github.io/siddhi-io-kafka/api/latest/)**
-
-## Removed features and functionalities
-
-These features are unsupported and removed from WSO2 API Manager 4.1.0 onwards.
-
-- **DAS Message Tracer**
-
-     The message tracer feature for WSO2 DAS is no longer supported and removed from the product.
-
-- **Hazelcast Based Gateway Clustering**
-
-     The Hazelcast based gateway clustering is no longer supported and removed from the product.
+??? note "Introducing MI control plane, Search and Pagination"
+    This feature introduces a new Micro Integrator Control plane and Implement Pagination and Search Functionality.
 
 
 ## Compatible WSO2 product versions
 
-- WSO2 API Manager 4.1.0 is compatible with WSO2 Identity Server 5.11.0.
-- WSO2 API Manager 4.1.0 is compatible with Choreo Connect {{choreo_connect.version}}.
+- WSO2 API Manager 4.2.0 is compatible with WSO2 Identity Server 6.0.0.
+- WSO2 API Manager 4.2.0 is compatible with Choreo Connect {{choreo_connect.version}}.
```

Comment:
I don't think this is correct because APK is coming in and CC is going out of the docs. Let's check with Pubudu and remove.

Reply:
Keeping this, because we have not removed CC docs yet. Let this be as a placeholder. WDYT?

Reply:
Ok, +1. We can change this once removed.